shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -182,6 +183,117 @@
   return canvases;
 }
 
+int FlutterPlatformViewsController::GetNumberOfClips(const MutatorsStack& mutators_stack) {
+  // Apply transforms/clips.
+  std::vector<std::shared_ptr<Mutator>>::const_reverse_iterator iter = mutators_stack.bottom();
+  int clipCount = 0;
+  while (iter != mutators_stack.top()) {
+    if ((*iter)->isClipType()) {
+      clipCount++;
+    }
+    ++iter;
+  }
+  return clipCount;
+}
+
+UIView* FlutterPlatformViewsController::ReconstructClipViewsChain(int number_of_clips,
```

Comment:
I'm not sure what `child` and `parent` mean in this context... would it make sense to call it `platform_view` and `head_clip_view`?